src/app/visual/datepicker/datepicker-visual.component.ts

```diff
@@ -60,12 +60,6 @@ export class DatepickerVisualComponent implements OnInit {
   }
 
   public toggleDisabled(): void {
-    if (this.reactiveDate.disabled) {
```

Comment:
Since this is the proper way to disable a reactive form, I would put this `if/else` condition back into the `toggleDisabled` function to make sure the two methods work well in tandem.

Reply:
I checked that it does, I took it out of the demo because I wasn't sure if we wanted people to just use the `disabled` property or not. I'll add it back!